src/imputation.py

```diff
@@ -0,0 +1,109 @@
+import traceback
+import Algorithmia
+import pandas as pd
+import os
+
+client = Algorithmia.client()
+
+
+def apply(input):
+
+    # API calls will begin at the apply() method, with the request body passed as 'input'
+    # For more details, see algorithmia.com/developers/algorithm-development/languages
+
+    try:
+        result = do_imputation()
+    except Algorithmia.errors.AlgorithmException as exc:
+        return {
+            "success": False,
+            "error": _get_traceback(exc)
+        }
+    except Exception as exc:
+        return {
+            "success": False,
+            "error": "Unexpected exception {}".format(_get_traceback(exc))
+        }
+
+    output = result.to_csv(index=False)
+
+    # Algorithmia.client().file("data://glanvl/BMIData/output_Lamb_Test.csv").put(str(output))
+    print("Stage: Final Output Sent")
+    return {
+            "success": True,
+            "data": output  # .to_json(orient='records')[1:-1].replace('}, {','},{')
+            }
+
+
+def do_imputation():
+    print("Stage: Running do_imputation")
+    period = 201809
+    cwd = os.path.dirname(__file__)
+    imported_data = pd.read_csv('{}/tests/fixtures/test_data_imputation.csv'.format(cwd), index_col=False)
+    responder_data = check_non_response(imported_data, period)
+    print(str(responder_data))
+
+    return responder_data  # .to_json(orient='records')
+
+
+def check_non_response(data, period):
+    print("Stage: Running check_non_response")
```

Comment:
we should lose the print statements which are really from you guys developing / debugging